src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1933,6 +1931,9 @@ private void PowerShellContext_ExecutionStatusChangedAsync(object sender, Execut
             var options = e?.ExecutionOptions;
             if (options == null || !options.IsReadLine)
             {
+                // TODO: This should somehow check if the server has actually started because we are
+                // currently sending this notification before it has initialized, which is not
+                // allowed. We should also figure out exactly _why_ that's happening.
```

Comment:
@rjmholt FYI this is what we see in the logs. IDK if it's what's _breaking_ the startup though.